envergo/templates/pages/geometricians.html

```diff
@@ -0,0 +1,204 @@
+          <br />
+          <span class="fr-text fr-text--sm">3 minutes, gratuit et anonyme</span>
+        </div>
+        <div id="geometrician-illustration" class="fr-col-12 fr-col-lg-4">
+          <img src="{% static 'images/geometrician.svg' %}"
+               alt="Illustration d'un géomètre">
+        </div>
+      </div>
+    </div>
+    <svg id="rounded-hero" viewBox="0 0 800 140">
+      <path fill="#fff" fill-opacity="1" d="M 0 100 Q 350 200 800 0 L 800 200 L 0 200 L 0 100 "></path>
+    </svg>
+
+  </section>
+
+  <section id="advantages" class="fr-container">
+    <div class="fr-grid-row fr-grid-row--gutters fr-py-5w">
+      {% for title, description, light_picture_src, dark_picture_src in advantages %}
+        <div class="fr-col fr-col-12 fr-col-md-4">
+          <img src={% static light_picture_src %} alt={{ description }} class="light-only">
+          <img src={% static dark_picture_src %} alt={{ description }} class="dark-only">
+          <h2>{{ title }}</h2>
+          <p class="fr-text fr-mb-2w">{{ description }}</p>
+        </div>
+      {% endfor %}
+    </div>
+  </section>
+
+  <section id="simulateur"
+           class="moulinette moulinette-columns fr-pt-8w fr-pb-5w">
+    <div class="fr-container">
+      <h2 class="fr-h3 fr-mb-6w">Simulez votre projet !</h2>
+      <div class="fr-grid-row fr-grid-row--center">
+        <div class="fr-col fr-col-lg-9">{% include 'moulinette/_form.html' with form=form %}</div>
+      </div>
+    </div>
+  </section>
+
+  <section class="fr-container fr-pt-8w fr-pb-5w">
+    <h2 class="fr-h3 fr-mb-6w">Témoignages</h2>
+    <div class="fr-grid-row fr-grid-row--center">
+      {% for testimony, title, department in testimonials %}
+        <div class="fr-col fr-col-12 fr-col-md-4 fr-pr-4w fr-mb-4w">
+          <figure class="fr-quote">
+            <blockquote>
+              <p>« {{ testimony }} »</p>
+            </blockquote>
+            <figcaption>
+              <ul class="fr-quote__source">
+                <li>{{ title }}</li>
+                <li>{{ department }}</li>
+              </ul>
+            </figcaption>
+          </figure>
+        </div>
+      {% endfor %}
+    </div>
+  </section>
+
+  <section id="webinar" class="fr-pt-8w fr-pb-5w">
+    <div class="fr-container">
+      <h2 class="fr-h3 fr-mb-2w">Participez à notre webinaire mensuel dédié aux géomètres</h2>
+      <div class="fr-grid-row fr-grid-row--middle">
+        <div id="webinar-img-container"
+             class="fr-col fr-col-12 fr-col-md-6 fr-pr-2w">
+          <img src="{% static 'images/webinar.jpg' %}"
+               alt="screenshot of a webinar session">
+        </div>
+        <div class="fr-col fr-col-12 fr-col-md-6 fr-pl-2w fr-mt-4w">
+
+          <p class="fr-mb-2w">20 minutes pour :</p>
+          <ul class="fr-mb-3w">
+            <li>Découvrir le service avec une démonstration du simulateur</li>
+            <li>Retours d'expériences de géomètres utilisateurs.trices</li>
```

Comment:
Il me semble que le point média « · » est plus adapté dans ces cas là.

```suggestion
            <li>Retours d'expériences de géomètres utilisateurs·trices</li>
```